extensions/sql-migration/src/dashboard/sqlServerDashboard.ts

```diff
@@ -219,26 +232,47 @@ export class DashboardWidget {
 		this._viewAllMigrationsButton.enabled = false;
 		this._migrationStatusCardLoadingContainer.loading = true;
 		try {
-			this._migrationStatus = await this.getMigrations();
-
-			const inProgressMigrations = this._migrationStatus.filter((value) => {
+			this.setCurrentMigrations(await this.getMigrations());
+			const migrationStatus = await this.getCurrentMigrations();
+			const inProgressMigrations = migrationStatus.filter((value) => {
 				const status = value.migrationContext.properties.migrationStatus;
 				const provisioning = value.migrationContext.properties.provisioningState;
 				return status === 'InProgress' || status === 'Creating' || status === 'Completing' || provisioning === 'Creating';
```

Comment:
You use these in a couple places, it would be good to have an enum type for the known statuses